CHANGES.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 
 Unreleased
 -----------
+- Add options to control free variables in Examples and step definitions
```

Comment:
yeah.. what's free? is it all about checking the unused variables or not?

Reply:
It's just about terminology) Free == unused
https://en.wikipedia.org/wiki/Free_variables_and_bound_variables